### PR TITLE
Update Batchnorm test

### PR DESCRIPTION
**Pull request by lara-hdr**
Updating batchnorm tests to succed with ORT backend.

### CODE REVIEW

File: docs/Operators.md [hunk 1]
```diff
@@ -1969,7 +1969,9 @@ s = np.array([1.0, 1.5]).astype(np.float32)
 bias = np.array([0, 1]).astype(np.float32)
 mean = np.array([0, 3]).astype(np.float32)
 var = np.array([1, 1.5]).astype(np.float32)
-training_mode = np.ones(1, dtype=bool)
+# using np.bool(1) while generating test data with "'bool' object has no attribute 'dtype'"
+# working around by using np.byte(1).astype(bool)
+training_mode = np.byte(1).astype(bool)
```
Comment (shinh) [inline docs/Operators.md]: `np.ones((), dtype=bool)` or `np.array(True)` would also work, this is also fine, though?

**Comment by lara-hdr** [comment]
@wschin @prasanthpul @faxu

**Comment by lara-hdr** [comment]
@wschin @prasanthpul @faxu

File: docs/Operators.md (hunk 1 above)
Comment (lara-hdr) [reply docs/Operators.md]: Good to know, thanks!
Yes I tested with np.byte(1).astype(bool) and it works as well

**Comment by lara-hdr** [comment]
@wschin could you review/approve please? thanks!

**Comment by lara-hdr** [comment]
@wschin @prasanthpul @faxu @ebarsoum.
This PR is for fix the tests running in ORT, please review!
Thanks

**Pull request merged**